nestris_ocr/utils/lib.py

```diff
@@ -2,6 +2,10 @@ def lerp(start, end, perc):
     return perc * (end - start) + start
 
 
+def ilerp(start, end, perc):
```

Comment:
What does the rounding achieve?

Reply:
Just being OCD for precision. Without the structure transform done with `np.array(dtype=np.uint8)` always `floors()`, and so a channel value interpolated at 123.9 will become 123, while it should be 124

Very very minor consideration 😅

Reply:
Ahh ok. I'd be interested slightly in performance but i suspect there is 0 impact.

Reply:
Yeah, I think so too. Especially in fastest where colors will be interpolated at most once per level.